docs/general/installation/container.md

```diff
@@ -273,3 +273,33 @@ As always it is recommended to run the container rootless. Therefore we want to
    ```sh
    systemctl --user enable --now podman-auto-update.timer
    ```
+
+## TrueNAS SCALE / TrueCharts
+
+Jellyfin is available as a [TrueNAS SCALE](https://www.truenas.org/) App inside the [TrueCharts](https://www.truecharts.org/) App Catalog with direct intergation into the GUI, no CLI needed. Direct support is available on the [TrueCharts Discord](https://discord.com/Js6xv9nGuU) and the source code is available on [GitHub](https://github.com/truecharts/charts).
+
+1. Install the TrueCharts Catalog to TrueNAS SCALE, see [website](https://truecharts.org/manual/guides/Adding-TrueCharts) for more info.
+
+   ```sh
```

Comment:
Why not use a list in a list instead of a codeblock? (same for step 3)

Reply:
Just thought it looked better, but I'll reformat